packages/manager/apps/dedicated/client/app/account/contacts/service/user-contacts.constants.js

```diff
@@ -0,0 +1,27 @@
+export const AVAILABLE_SERVICES = [
+  'CLOUD',
+  'DOMAIN',
+  'HOSTING',
+  'HOSTING_RESELLER',
+  'PRIVATE_DATABASE',
+  'EMAIL_DOMAIN',
+  'DEDICATED',
+  'KUBE',
+  'NAS',
+  'NASHA',
+  'OVER_THE_BOX',
+  'OVH_CLOUD_CONNECT',
+  'PRIVATE_CLOUD',
+  'ESSENTIALS',
+  'VPS',
+  'PACK_XDSL',
+  'XDSL',
+  'ZONE',
+  'FAILOVER',
+  'PACK_SIP_TRUNK',
+  'LOAD_BALANCER',
```

Comment:
i think your last rebase did not go well. it replaced all the changes made. I added "Logs" entery here which is missing.